em_plugins/webview/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "webview-background",
+  "version": "0.0.2",
+  "description": "A small plugin to change the background of a webview.",
+  "main": "dist/plugin.cjs.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "unpkg": "dist/plugin.js",
+  "files": [
+    "android/src/main/",
+    "android/build.gradle",
+    "dist/",
+    "ios/Sources",
+    "Package.swift",
+    "WebviewBackground.podspec"
+  ],
+  "author": "Matheu DeWitt",
+  "license": "MIT",
```

Comment:
Please omit the license so that it is included under the parent package's license.